module/documents/chat-message.mjs

```diff
@@ -291,7 +298,7 @@ export default class ChatMessage5e extends ChatMessage {
           </div>
           <div class="ac">
             <i class="fas fa-shield-halved"></i>
-            <span>${ac}</span>
+            <span>${displayedAC}</span>
           </div>
```

Comment:
It would be good to remove this whole block that way players don't see just a random shield icon:

```javascript
          ${ac ? `
          <div class="ac">
            <i class="fas fa-shield-halved"></i>
            <span>${ac}</span>
          </div>
          ` : ""}
```